src/frontend/src/icp-eth/components/tokens/AddTokenByNetwork.svelte

```diff
@@ -0,0 +1,40 @@
+<script lang="ts">
+	import { i18n } from '$lib/stores/i18n.store';
+	import { Dropdown, DropdownItem } from '@dfinity/gix-components';
+	import { networksMainnets } from '$lib/derived/networks.derived';
+	import { ETHEREUM_NETWORK_SYMBOL, ICP_NETWORK_SYMBOL } from '$env/networks.env';
+	import IcAddTokenForm from '$icp/components/tokens/IcAddTokenForm.svelte';
+	import AddTokenForm from '$eth/components/tokens/AddTokenForm.svelte';
+	import { networkId } from '$lib/derived/network.derived';
+	import { isNullish } from '@dfinity/utils';
+
+	export let dropdownNetwork: string | undefined;
```

Comment:
- any particular reason to prefix this with "dropdown" keyword? we generally do not use UI component prefix sofar if I'm not wrong
- any particular reason to use a `string` instead of using the network id?

Reply:
- i wanted to call it `selectedNetwork`, but I didn't want it to confound it with the derived `selectedNetwork`, do you have some idea of a better name?
- I tried symbol, but `Dropdown` component accepts only string as `selectedValue` prop and it can be bound only to the real variable not the prop: we cannot do `bind:selectedValue={dropdownNetwork.description}`

Reply:
- `managedNetwork` or `customTokensNetwork`?
- Ah yeah right. I really don't say it's a good pattern but, should we do something similar to `SendNetwork` component using `name`?

Reply:
Actually isn't the feature really close to what the `SendNetwork` does, is there a chance to have or reuse a same component for both or that's too different? Just an idea that crossed my mind right now, not sure it's a good idea...

Reply:
I was looking at it right now

I would prefer to use the same, but it has some variables that are specific to the Send process (`destination` for example) and some function too (onDestinationAddressInput for example)

I can adapt this one to use the same logic for selection thoough... (and same format, style, strings, etc)

Reply:
Sounds cool, thanks

Reply:
done